### PR TITLE
updates signing modal to be more helpful for ledger and metamask users

### PR DESCRIPTION
**Pull request by huesforalice**

### CODE REVIEW

File: app/modules/settings/sagas.ts [hunk 1]
```diff
@@ -31,43 +26,20 @@ export function* addNewEmail(
...
-  const walletType = yield select((s: IAppState) => selectWalletType(s.web3));
   const salt = yield select(
     (s: IAppState) => selectLightWalletSalt(s.web3) || selectPreviousLightWalletSalt(s.web3),
   );
   const isEmailAvailable = yield select((s: IAppState) => selectDoesEmailExist(s.auth));
 
-  const emailModalTitle = isEmailAvailable ? "Email Update" : "Add Email";
-
-  let addEmailMessage;
+  const emailModalTitle = isEmailAvailable ? "Update Email" : "Add Email";
```
Comment (trival) [inline app/modules/settings/sagas.ts]: here are still untranslated strings

Comment (huesforalice) [reply app/modules/settings/sagas.ts]: done

File: app/components/modals/walletAccess/AccessWalletModal.tsx [hunk 2]
```diff
@@ -3,51 +3,84 @@ import * as React from "react";
...
-import { selectIsLightWallet, selectIsUnlocked } from "../../../modules/web3/selectors";
+import { selectIsUnlocked, selectWalletType } from "../../../modules/web3/selectors";
 import { appConnect } from "../../../store";
 import { TTranslatedString } from "../../../types";
 import { ModalComponentBody } from "../ModalComponentBody";
 import { AccessLightWalletPrompt } from "./AccessLightWalletPrompt";
 
+import * as ledgerConfirm from "../../../assets/img/wallet_selector/ledger_confirm.svg";
 import * as lockIcon from "../../../assets/img/wallet_selector/lock_icon.svg";
+import { EWalletType } from "../../../modules/web3/types";
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/modals/walletAccess/AccessWalletModal.tsx]: done

File: app/components/modals/walletAccess/AccessWalletModal.tsx [hunk 3]
```diff
@@ -3,51 +3,84 @@ import * as React from "react";
...
+      <div>
+        <img src={lockIcon} className="mt-3 mb-3" />
+        <AccessLightWalletPrompt onAccept={onAccept} isUnlocked={isUnlocked} />
+      </div>
+    )}
+    {walletType === EWalletType.LEDGER && (
+      <div>
+        <img src={ledgerConfirm} className="mt-1 mb-3" />
+        <div className={cn("mt-2", styles.info)}>
+          Please confirm this action with your Ledger Nano!
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/modals/walletAccess/AccessWalletModal.tsx]: good find, thanks, now donw

**Pull request merged**